### PR TITLE
Shs 108 name overlap mobile

### PR DESCRIPTION
**Pull request by ryanhagerty**
## Summary
Overrides some inherited styles from an external 'decanter' component. The original styles would allow the 'School of...' text to overflow onto the menu button.

## Steps to Test
1. This is a good candidate for Tugboat testing as it doesn't need any new pages. 
2. Go to the homepage from the link below and go down to mobile size in browser and make sure the text doesn't overflow onto the 'Menu' button.
3. Do this for both colorful/traditional

## Screenshots
Screen Shot 2022-09-27 at 1 53 04 PM

## Notes
The original and real issue with this is that menu button is outside of document flow. It is `absolute` and then pulled up with a negative `top`. This is less than ideal and leaves an opening for more potential wonkiness in the future. 

I spent some time refactoring the menu nav button to be in proper DOM order, however the changes were getting extensive, touched many files, and the JS functionality for controlling the open/close state of the menu started to behave inconsistently. Unfortunately, to do this the proper way I think we need to factor the entire V1 mega menu. I preserved the work I was doing in a branch, but I think the time vs. benefit here ended up going beyond the ticket scope.

Leaving base branch as `develop` until new sprint branch.

### CODE REVIEW

**Comment by su-sws-jira** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Links:
- https://pr1188-ijoc4ch7irdelig8l4ruruxvt6eyz9av.tugboatqa.com
- https://hs-colorful-ijoc4ch7irdelig8l4ruruxvt6eyz9av.tugboatqa.com
- https://hs-traditional-ijoc4ch7irdelig8l4ruruxvt6eyz9av.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/63333a6012a495ddd4c52d36

File: docroot/themes/humsci/humsci_basic/src/scss/components/_masthead.scss [hunk 1]
```diff
@@ -4,7 +4,21 @@
   border-bottom: $hb-thin-border;
   @include hb-global-color('border-color', 'gray');
 
+  .su-site-header-nav-container {
+    display: grid;
```
Comment (callinmullaney) [inline docroot/themes/humsci/humsci_basic/src/scss/components/_masthead.scss:8]: :clap: Nice! This looks like it has the same affect I proposed in the ticket. Good call refactoring this a bit more.

**Comment by su-sws-jira** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Links:
- https://pr1188-ijoc4ch7irdelig8l4ruruxvt6eyz9av.tugboatqa.com
- https://hs-colorful-ijoc4ch7irdelig8l4ruruxvt6eyz9av.tugboatqa.com
- https://hs-traditional-ijoc4ch7irdelig8l4ruruxvt6eyz9av.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/63333a6012a495ddd4c52d36

File: docroot/themes/humsci/humsci_basic/src/js/shared/navigation/change-nav.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 const changeNav = (toggle, parent, isOpen) => {
   toggle.setAttribute('aria-expanded', isOpen);
   parent.setAttribute('aria-hidden', !isOpen);
+  console.log(parent);
```
Comment (callinmullaney) [inline docroot/themes/humsci/humsci_basic/src/js/shared/navigation/change-nav.js]: Looks like this console.log slipped into a commit. I'll remove it.

**Review by callinmullaney — APPROVED** [review]
@ryanhagerty This PR is looking good to merge. I pulled develop, fixed conflicts, and removed a `console.log` that made it's way into a previous commit.

Thanks for attempting to refactor that header structure a bit more. When I looked into the technical details of this ticket I also noticed the menu toggle was positioned absolute. As you discovered with attempting to fully refactor the header, the task became a bit more complicated. 

Going forward I'll attempt to be more explicit about "this thing needs refactoring" or "lets implement a quick fix and refactor later". 

color and traditional looked good. I also reviewed datasciencemajor locally to verify this longer name was inheriting these changes properly.
Screenshot from 2022-09-30 14-27-12

**Comment by su-sws-jira** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Links:
- https://pr1188-ijoc4ch7irdelig8l4ruruxvt6eyz9av.tugboatqa.com
- https://hs-colorful-ijoc4ch7irdelig8l4ruruxvt6eyz9av.tugboatqa.com
- https://hs-traditional-ijoc4ch7irdelig8l4ruruxvt6eyz9av.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/63333a6012a495ddd4c52d36

**Comment by su-sws-jira** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Links:
- https://pr1188-ijoc4ch7irdelig8l4ruruxvt6eyz9av.tugboatqa.com
- https://hs-colorful-ijoc4ch7irdelig8l4ruruxvt6eyz9av.tugboatqa.com
- https://hs-traditional-ijoc4ch7irdelig8l4ruruxvt6eyz9av.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/63333a6012a495ddd4c52d36

**Pull request merged**